Index: redis-embedded/src/main/java/com/lukeshannon/RedisTemplateConfig.java
===================================================================
--- redis-embedded/src/main/java/com/lukeshannon/RedisTemplateConfig.java	(revision 91aac0ceefaf361b499a0a666b67ae947d37f7a9)
+++ redis-embedded/src/main/java/com/lukeshannon/RedisTemplateConfig.java	(revision 91aac0ceefaf361b499a0a666b67ae947d37f7a9)
@@ -1,18 +0,0 @@
-package com.lukeshannon;
-
-import org.springframework.context.annotation.Bean;
-import org.springframework.context.annotation.Configuration;
-import org.springframework.data.redis.connection.RedisConnectionFactory;
-import org.springframework.data.redis.core.RedisTemplate;
-
-@Configuration
-public class RedisTemplateConfig {
-	
-	@Bean
-	RedisTemplate<?, ?> redisTemplate(RedisConnectionFactory connectionFactory) {
-		RedisTemplate<byte[], byte[]> template = new RedisTemplate<>();
-		template.setConnectionFactory(connectionFactory);
-		return template;
-	}
-
-}
Index: redis-embedded/src/main/java/com/lukeshannon/RedisTestConfiguration.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- redis-embedded/src/main/java/com/lukeshannon/RedisTestConfiguration.java	(revision 91aac0ceefaf361b499a0a666b67ae947d37f7a9)
+++ redis-embedded/src/main/java/com/lukeshannon/RedisTestConfiguration.java	(date 1519198252000)
@@ -2,8 +2,8 @@
 
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
-import org.springframework.boot.test.context.TestConfiguration;
 import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
 import org.springframework.context.annotation.Profile;
 import org.springframework.data.redis.connection.RedisConnectionFactory;
 import org.springframework.data.redis.connection.lettuce.LettuceConnectionFactory;
@@ -11,14 +11,14 @@
 /*
  * https://docs.spring.io/spring-boot/docs/current/reference/html/boot-features-testing.html#boot-features-testing-spring-boot-applications-detecting-config
  */
-@TestConfiguration
+@Configuration
 @Profile("test")
 public class RedisTestConfiguration {
-	
+
 	private static final Logger log = LoggerFactory.getLogger(RedisTestConfiguration.class);
-	
+
 	@Bean
-	RedisConnectionFactory connectionFactory(EmbeddedRedis embeddedRedisServer) {
+	RedisConnectionFactory redisConnectionFactory(EmbeddedRedis embeddedRedisServer) {
 		log.info("Creating a Redis Connection Factory to embedded Redis");
 		return new LettuceConnectionFactory("127.0.0.1", embeddedRedisServer.getRedisPort());
 	}
Index: redis-embedded/pom.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- redis-embedded/pom.xml	(revision 91aac0ceefaf361b499a0a666b67ae947d37f7a9)
+++ redis-embedded/pom.xml	(date 1519198183000)
@@ -37,9 +37,21 @@
 		<dependency>
 			<groupId>org.springframework.boot</groupId>
 			<artifactId>spring-boot-starter-test</artifactId>
-			<scope>test</scope>
+			<scope>compile</scope>
+		</dependency>
+
+		<dependency>
+			<groupId>biz.paluch.redis</groupId>
+			<artifactId>lettuce</artifactId>
+			<version>4.4.3.Final</version>
+		</dependency>
+
+		<dependency>
+			<groupId>io.netty</groupId>
+			<artifactId>netty-all</artifactId>
+			<version>4.1.20.Final</version>
 		</dependency>
-		
+
 		<dependency>
 			<groupId>org.projectlombok</groupId>
 			<artifactId>lombok</artifactId>
@@ -51,9 +63,8 @@
 			<groupId>it.ozimov</groupId>
 			<artifactId>embedded-redis</artifactId>
 			<version>0.7.1</version>
-			<scope>test</scope>
 		</dependency>
-		
+
 		<!-- Starting PCF Connector Dependancies -->
 		<dependency>
 			<groupId>org.springframework.cloud</groupId>
@@ -68,7 +79,7 @@
 			<artifactId>spring-cloud-localconfig-connector</artifactId>
 		</dependency>
 		<!-- End PCF Connector Dependancies -->
-		
+
 
 
 	</dependencies>
